### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by GroovinChip**
Add note about potential IDE restart required

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,8 @@ Used by the [Dart Package site](https://pub.dev/).
 > pub global activate pana
 ```
 
+NOTE: You may need to restart your IDE for pana commands to be recognized.
```
Comment (sigurdm) [inline README.md:22]: Is this a general thing for pub global activate?

What ide did you experience this with?

**Comment by GroovinChip** [comment]
I experienced this with Android Studio

**Comment by sigurdm** [comment]
And how do you call pana from android studio? Does it have its own button? Or is it from the shell?

**Comment by GroovinChip** [comment]
It is from the shell

**Comment by sigurdm** [comment]
Usually bash picks up new binaries from `PATH` automatically without a restart - so I'm not sure why this is needed.

**Comment by GroovinChip** [comment]
The new text only says a restart *may* be needed

**Comment by sigurdm** [comment]
Hmm - so many things *may* be needed under different circumstances we cannot enumerate them all.

If a restart of the IDE **is** needed I think the advice is very helpful.

I would like to understand the mechanism at play here better before including more instructions here. Then we can also make more precise instructions. Maybe we should write: 

> NOTE: You may need to restart your shell for pana commands to be recognized.

But without understanding why bash doesn't pick this up, it is hard to recommend something specific.

**Comment by GroovinChip** [comment]
Yeah, I don't know what would cause the problem so a generic statement like
that would be helpful to me, personally. I can change it to your suggestion
of you like?

**Pull request closed without merging**